README.md

```diff
@@ -42,19 +43,38 @@ health-checker [options]
 
 | Option | Description | Default 
 | ------ | ----------- | -------
-| `--port` | The port number on which a TCP connection will be attempted. Specify one or more times. | | 
+| `--port` | The port number on which a TCP connection will be attempted. Can be specified multiple times. | |
+| `--http` | The url:port to check for a 2XX status code. Can be specified multiple times. | |
```

Comment:
Hm, should we support other status codes that count as success? For example, Vault has several different status codes: https://www.vaultproject.io/api/system/health.html.

Similarly, does it make sense to be able to scan the body of the HTTP response for some regex? Might be another good, common sanity check. 

For clarity, we don't need to support all of these features in the very next iteration, but we should think through an API / UX that makes those possible.

Reply:
Also, we should mention if HTTPS URLs will work here.

Reply:
re: different status codes - I was thinking about possibly allowing anything but a 5XX as success. This can get messy though, even ELB vs ALB has this issue - for ELBs 3XX is considered healthy where ALB only reports healthy for 2XX. 

That said, as I think about this - it makes more sense for now to just assume success for any status code other than a 5XX/4XX - the reason I am bundling 4XX is mainly because of 404 and 400 status codes.

Reply:
re: HTTPS - unless we know of a use case for this already - I'd like to just be more clear that it only supports HTTP at this time and track a separate feature/improvement if we have a need.